Client/Client.Qt/Widgets/PopupMessage.hpp

```diff
@@ -30,5 +48,5 @@ private slots:
     QGridLayout             _layout;
     QPropertyAnimation      _animation;
     std::unique_ptr<QTimer> _timer;
-    int                     _showTime = 3000;
+    int                     _animationDuration = 3000;
```

Comment:
You change the signature of the function, but forget about the field that keep time...
Don't be like that :(